src/graph/edge.cc

```diff
@@ -286,6 +286,7 @@ namespace hpp {
         constraints->configProjector ()->rightHandSideFromConfig(q1);
         if (constraints->isSatisfied (q1)) {
           if (constraints->isSatisfied (q2)) {
+	    if (q1.isApprox (q2)) return false;
```

Comment:
Does it make sense to do this check here ?

I would rather return a path of length zero and let the caller handle this case. It shouldn't be an error because, in this case, the transition is a success.

Independently of the answer, this conflicts with the automatic generation of the graph.